include/net/net_pkt.h

```diff
@@ -128,7 +128,7 @@ struct net_pkt {
 	};
```

Comment:
Typo in commit message of "net: ipv6: Fix HBHO parsing options function"

s/parsing is failed/parsing fails/
s/declared as/is declared as/

The "PAD1 contains only one byte which is 0." looks confusing, could we say here "PAD1 contains only one byte which has value 0."